### PR TITLE
add openssh root login troubleshooting

### PR DESCRIPTION
**Pull request by GavinFrazar**
This is a docs PR that adds a troubleshooting section for agentless openssh connections as root.

A customer had an issue where they could only login to their agentless openssh server as non-root users.
We discovered they had `PermitRootLogin no` in their sshd config.
The customer had no issue connecting as root to servers that were running a Teleport ssh agent, hence the confusion.

Maybe this section will help someone in the future 🤷‍♂️

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/server-access/troubleshooting-server.mdx [hunk 1]
```diff
@@ -140,4 +140,17 @@ spec:
 ```
 
 For more information about moderated sessions and session sharing, see 
-[Moderated Sessions](../access-controls/guides/moderated-sessions.mdx).
\ No newline at end of file
+[Moderated Sessions](../access-controls/guides/moderated-sessions.mdx).
+
+## Unable to connect to Agentless OpenSSH server as root
```
Comment (ptgott) [inline docs/pages/server-access/troubleshooting-server.mdx:145]:
```suggestion
## Unable to connect to agentless OpenSSH server as root
```

File: docs/pages/server-access/troubleshooting-server.mdx [hunk 2]
```diff
@@ -140,4 +140,17 @@ spec:
...
+
+You should check sshd configuration in /etc/ssh/sshd_config for a setting like
```
Comment (ptgott) [inline docs/pages/server-access/troubleshooting-server.mdx:147]:
```suggestion
You should check your sshd configuration in `/etc/ssh/sshd_config` for a setting like
```

File: docs/pages/server-access/troubleshooting-server.mdx [hunk 3]
```diff
@@ -140,4 +140,17 @@ spec:
...
+`PermitRootLogin no` or `PermitRootLogin forced-commands-only` - either of these
+settings will prevent login as root. 
+If you wish to login as root to an OpenSSH server via Teleport, we recommend
```
Comment (ptgott) [inline docs/pages/server-access/troubleshooting-server.mdx:150]:
```suggestion
If you wish to log in as root to an OpenSSH server via Teleport, we recommend
```
"log in" is the verb phrase

**Review by ptgott — APPROVED** [review]
Approved with some grammar nits

**Comment by ptgott** [comment]
Should we backport this one?

File: docs/pages/server-access/troubleshooting-server.mdx (hunk 3 above)
Comment (GavinFrazar) [reply docs/pages/server-access/troubleshooting-server.mdx:150]: always gets me! thanks

**Comment by GavinFrazar** [comment]
> Should we backport this one?

Yeah I will

**Pull request merged**